// #pragma once

// #include <cstdint>
// #include <fcitx-utils/eventdispatcher.h>
// #include <fcitx-utils/inputbuffer.h>
// #include <fcitx/addonfactory.h>
// #include <fcitx/addonmanager.h>
// #include <fcitx/candidatelist.h>
// #include <fcitx/inputcontext.h>
// #include <fcitx/inputcontextproperty.h>
// #include <fcitx/inputmethodengine.h>
// #include <fcitx/inputpanel.h>
// #include <fcitx/instance.h>
// #include <memory>
// #include <vector>

